app/model.py

```diff
@@ -649,9 +653,13 @@ def put_new(self):
         a new note is created. Also, logs user actions is updated. We should
         never call this method against an existing record."""
         db.put(self)
-        UsageCounter.increment_counter(self.repo, 'note')
         UserActionLog.put_new('add', self, copy_properties=False)
-
+        for note_status in NOTE_STATUS_TEXT:
```

Comment:
I believe you don't need this for-loop. You can just directly use self.status instead of note_status.